packages/react-server-examples/bike-share/pages/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {ReactServerAgent, RootContainer, RootElement} from 'react-server';
+import NetworkList from '../components/network-list';
+import Header from '../components/header';
+import Footer from '../components/footer';
+
+export default class IndexPage {
+	handleRoute(next) {
+		this.data = ReactServerAgent.get('http://api.citybik.es/v2/networks');
+		return next();
+	}
+	getElements() {
+		return [
+			<RootContainer key={0}>
```

Comment:
I don't _think_ those warnings should arise from the root level, since we don't actually let React render root containers or the outer array.  Would be nice to keep the example JSX clean of extraneous attributes.